lib/note-list/style.scss

```diff
@@ -114,6 +114,11 @@
     color: transparent;
     height: 16px;
     width: 16px;
+
+    .icon-pinned-small {
+      height: 16px;
```

Comment:
Again - these sizes should inherit properly from the parent, it might need a `flex: 0 0 auto` to say "do not squish or stretch me"

Reply:
(and on the parent, probably, she says without testing it)

Reply:
I couldn't get this to work, I'm likely missing something, but I did come up with what I think is a better way to set the sizes and have updated it with that now.